regression-tests/backends/bind-master

```diff
@@ -59,7 +59,12 @@ __EOF__
 			fi
 			if [ $zone != insecure.dnssec-parent.com ]
 			then
-				securezone $zone bind
+                if [ $zone != hiddencryptokeys.org ]
+                then
+    				securezone $zone bind
+                else
+                    $PDNSUTIL --config-dir=. --config-name=bind add-zone-key $zone zsk 384 active unpublished ecdsa384
```

Comment:
mismatching indent.

Reply:
I've done a retab on the offending files to fix this.